core/controllers/base.py

```diff
@@ -278,6 +293,9 @@ def delete(self, *args):  # pylint: disable=unused-argument
         raise self.PageNotFoundException
 
     def render_json(self, values):
+        """ Decorator to render and writes data sent from the server
```

Comment:
There should be a similar docstring and args for render_template and other methods below...